packages/@aws-cdk/aws-appsync/test/appsync.auth.graphql

```diff
@@ -0,0 +1,12 @@
+type test {
+  id: Int!
+  version: String!
+}
+
+type Query {
+  getTests: [ test! ]
```

Comment:
How is this related to this change?

Reply:
just for the integrity test to see if the API Key works for query/mutation